### PR TITLE
add labkey upgrade lock file protection

### PR DESCRIPTION
**Pull request by labkey-stuartm**
- Move Java timezone to its own var for better handling elsewhere
- add LabKey upgrade lock file protection to prevent unexpected tomcat shutdowns while performing background upgrade tasks

### CODE REVIEW

File: install-labkey.bash [hunk 1]
```diff
@@ -1050,11 +1052,15 @@ function step_tomcat_service_embedded() {
 				Environment="LABKEY_JAR_OPS=${LABKEY_JAR_OPS}"
 				Environment="JAVA_LOG_JAR_OPS=${JAVA_LOG_JAR_OPS}"
 				Environment="JAVA_FLAGS_JAR_OPS=${JAVA_FLAGS_JAR_OPS}"
+				Environment="JAVA_TIMEZONE=${JAVA_TIMEZONE}"
+				Environment="LOCKFILE=${LOCKFILE}"
 				WorkingDirectory=${LABKEY_INSTALL_HOME}
 				OOMScoreAdjust=-500
 
-				ExecStart=$JAVA_HOME/bin/java \$JAVA_PRE_JAR_OPS \$JAVA_HEAP \$JAVA_MID_JAR_OPS \$LABKEY_JAR_OPS \$JAVA_LOG_JAR_OPS \$JAVA_FLAGS_JAR_OPS -jar ${LABKEY_INSTALL_HOME}/labkeyServer.jar
+				ExecStart=$JAVA_HOME/bin/java \$JAVA_TIMEZONE \$JAVA_HEAP \$JAVA_MID_JAR_OPS \$LABKEY_JAR_OPS \$JAVA_LOG_JAR_OPS \$JAVA_FLAGS_JAR_OPS -jar ${LABKEY_INSTALL_HOME}/labkeyServer.jar
```
Comment (labkey-willm) [inline install-labkey.bash]:
```suggestion
				ExecStart=$JAVA_HOME/bin/java \$JAVA_TIMEZONE \$JAVA_PRE_JAR_OPS \$JAVA_HEAP \$JAVA_MID_JAR_OPS \$LABKEY_JAR_OPS \$JAVA_LOG_JAR_OPS \$JAVA_FLAGS_JAR_OPS -jar ${LABKEY_INSTALL_HOME}/labkeyServer.jar
```

Comment (labkey-willm) [reply install-labkey.bash]: you left \$JAVA_PRE_JAR_OPS out of the new line.

**Review by labkey-willm — CHANGES_REQUESTED** [review]

File: install-labkey.bash (hunk 1 above)
Comment (labkey-stuartm) [reply install-labkey.bash]: eek!  Indeed I did!

Comment (labkey-stuartm) [reply install-labkey.bash]: fixed

**Review by labkey-willm — APPROVED** [review]

**Pull request merged**